README.md

```diff
@@ -53,4 +53,21 @@ Once the container is running, it should be accessible using:
 $ curl 127.0.0.1:5000
 ```
 
-Personally, I use a Mac to generate my code and docker image so I just open a browser to http://192.168.99.100:5000/`
+# Openshift Usage
+
+create the s2i builder image, dont need service or deployment
```

Comment:
Please start the sentence as uppercase (Create) and make it a complete sentence ("Create the s2i builder image, *you* don't need a service or deployment") - it's also a good idea to use a colon to indicate that you are describing the following `oc` commands